### DISCUSSION TITLE
How to switch IPs for web page

### DISCUSSION THREAD

**Discussion by jrg-merytronic** [cesanta/mongoose#2359] [Ask questions, get answers] [answered]
Hello,

I'm using Mongoose to create a little web, with LWIP and freeRTOS on STM32. The web page is working cool on a static IP. But this IP may change on air, due to DHCP configuration. I've been trying some solutions (like using open ip as 0.0.0.0 or using "mg_close_conn"), but noone is working. Is there any way to do this smoothly?

Thank you.

**Comment by scaprile** [comment, marked as answer]
Mongoose just binds to a socket, what does your network stack say you have to do when the IP changes ?
What _exactly_ are you doing ?
Please open an issue and follow the issue template
